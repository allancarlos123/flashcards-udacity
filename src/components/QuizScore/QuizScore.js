import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const QuizScore = ({
  score,
  questionsLength
}) => (
  <View>
    <Text style={styles.title}>Finished!</Text>
    <Text style={styles.subtitle}>{score} correct questions from {questionsLength}</Text>
  </View>
)

const styles = StyleSheet.create({
  title: {
    fontSize: 35,
    fontWeight: 'bold',
    color: '#ED4E5F'
  },
  subtitle: {
    fontSize: 24,
    textAlign: 'center'
  }
})

export default QuizScore