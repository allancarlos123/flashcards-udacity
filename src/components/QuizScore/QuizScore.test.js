import React from 'react'
import { create } from 'react-test-renderer'

import QuizScore from './'

describe('QuizScore', () => {
  const tree = create(
    <QuizScore
      score={40}
    />
  );

  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});