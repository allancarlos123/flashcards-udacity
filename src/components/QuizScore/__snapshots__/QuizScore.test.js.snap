// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizScore renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ED4E5F",
        "fontSize": 35,
        "fontWeight": "bold",
      }
    }
  >
    Finished!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 24,
        "textAlign": "center",
      }
    }
  >
    40
     correct questions from 
  </Text>
</View>
`;
